core/src/test/scala/unit/kafka/coordinator/group/GroupCoordinatorTest.scala

```diff
@@ -1222,6 +1200,7 @@ class GroupCoordinatorTest {
     assertEquals(Errors.NONE, followerSyncGroupResult._2)
     assertTrue(getGroup(groupId).is(Stable))
 
+    EasyMock.reset(replicaManager)
```

Comment:
By adding reset in the end of helper function, we reduced redundant replica manager reset.